### PR TITLE
Feat/create home page

### PR DESCRIPTION
**Pull request by NabeelahY**
# Description
I created and styled the home pages both for mobile and desktop view. I also seperated the code into various components to make it cleaner

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

**Review by Oyekunle-Mark — APPROVED** [review]
Great job Nabeelah. 👍 
I am a fan of your work. 🥇 🥇 🥇

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,70 @@
...
+    <div>
+      <GlobalStyles />
+      <Home />
+    </div>
+  );
 }
 
 export default App;
+
+const GlobalStyles = createGlobalStyle`
```
Comment (Anubhav311) [inline src/App.js:15]: Love this way of global styling.

File: src/index.js [hunk 2]
```diff
@@ -3,28 +3,33 @@ import React from 'react';
 import ReactDOM from 'react-dom';
```
Comment (Anubhav311) [inline src/index.js:3]: Good job with fixing the formatting. It gets really important really soon.

**Review by Anubhav311 — APPROVED** [review]
Good Job Nabeelah.

**Pull request merged**